src/libraries/System.Private.CoreLib/src/System/Globalization/RegionInfo.cs

```diff
@@ -63,7 +63,7 @@ public RegionInfo(int culture)
                 throw new ArgumentException(SR.Format(SR.Argument_CultureIsNeutral, culture), nameof(culture));
             }
 
-            if (culture == CultureInfo.LOCALE_CUSTOM_DEFAULT)
+            if (culture == CultureInfo.LOCALE_CUSTOM_DEFAULT || culture == CultureInfo.LOCALE_CUSTOM_UNSPECIFIED)
             {
                 // Not supposed to be neutral
                 throw new ArgumentException(SR.Format(SR.Argument_CustomCultureCannotBePassedByNumber, culture), nameof(culture));
```

Comment:
Is this the right exception message for `LOCALE_CUSTOM_UNSPECIFIED`? 

https://github.com/dotnet/runtime/blob/3ae87395f638a533f37b8e3385f6d3f199a72f4f/src/libraries/System.Private.CoreLib/src/Resources/Strings.resx#L970

Reply:
Yes, it still correct exception message.